client/src/components/BlogPostForm.tsx

```diff
@@ -43,6 +43,9 @@ export default function BlogPostForm({ entry }: props) {
   const [missingPhoto, setMissingPhoto] = useState(entry ? false : undefined);
   const [loadingAnimation, setLoadingAnimation] = useState(false);
 
+  const [userId] = useState(entry?.userId);
+  const [username] = useState(entry?.username);
```

Comment:
The useState in this situation is a placeholder for useContext information. I was going to bring in the signed in user's userId and username so that the entry they create can be attributed to them.

Reply:
Ah. That makes sense.